data/coalition/coalition outfits.txt

```diff
@@ -224,6 +224,17 @@ outfit "Scanning Module"
 	"tactical scan power" 25
 	description "This system allows the Heliarchs to scan nearby ships, enabling them to detect weapons or illegal goods."
 
+outfit "Outskirts Gauger"
+	category "Systems"
+	cost 1406000
+	thumbnail "outfit/outskirts gauger"
+	"mass" 8
+	"outfit space" -8
+	"tactical scan power" 256
+	"energy consumption" .25
+	"energy capacity" -200
+	description "Completed with the data from one of their long-lost satellites, the newest scanner of House Idriss excels at reading various energy signatures across vast distances."
+	description "	Constantly running its calculations about every little thing it detects within its impressive range, the gauger needs a steady supply of energy to function, as well as a set amount of battery capacity dedicated to it."
```

Comment:
```suggestion
	description "Constantly running its calculations about every little thing it detects within its impressive range, the gauger needs a steady supply of energy to function, as well as a set amount of battery capacity dedicated to it."
```

Reply:
Like with conversations, only the first `description` isn't indented, so this should be as it is.

Reply:
Sorry then, I'll delete this.